src/test/java/org/kiwiproject/champagne/dao/DeploymentEnvironmentDaoTest.java

```diff
@@ -88,26 +84,22 @@ class UpdateUser {
         @Test
         void shouldUpdateDeploymentEnvironmentSuccessfully(SoftAssertions softly) {
             long envId = insertDeploymentEnvironmentRecord(handle, "TEST", testUserId);
-            long testSecondUser = insertUserRecord(handle, "jadoe");
 
             var envToUpdate = DeploymentEnvironment.builder()
                     .id(envId)
                     .name("PRODUCTION")
-                    .updatedById(testSecondUser)
                     .build();
 
             dao.updateEnvironment(envToUpdate);
 
-            var users = handle.select("select * from deployment_environments where id = ?", envId)
+            var envs = handle.select("select * from deployment_environments where id = ?", envId)
                 .map(new DeploymentEnvironmentMapper())
                 .list();
 
-            assertThat(users).hasSize(1);
+            assertThat(envs).hasSize(1);
 
-            var user = first(users);
+            var user = first(envs);
             softly.assertThat(user.getName()).isEqualTo("PRODUCTION");
```

Comment:
No need for soft delete anymore...and don't forget to remove the argument and the extension itself